README.md

```diff
@@ -197,6 +197,12 @@ Docker can be used to manage and run SurrealDB database instances without the ne
 docker run --pull --rm -p 8000:8000 surrealdb/surrealdb:latest start
 ```
 
+If you expect errors like `No such image: surrealdb/surrealdb:latest` please try to `pull` the latest image first.
```

Comment:
Shouldn't it be "experience" not "expect"?

Reply:
```suggestion
If you experience errors like `No such image: surrealdb/surrealdb:latest` please try to just `pull` the latest image first.
```